CHANGES.md

```diff
@@ -34,6 +34,9 @@ pages**.
 for GScan, preventing that all suites are expanded after a single suite
 was expanded in GScan.
 
+[#3538](https://github.com/cylc/cylc-flow/pull/3532) - Fix job submission to
```

Comment:
I think the link is wrong, but everything else looks good.  :+1:

Reply:
Updated.